src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -520,11 +519,21 @@ class indexFieldOptions {
         });
     }
 
+    addCustomAnalyzers(customAnalyzers: string[]) {
+        customAnalyzers.forEach(name => {
+            if (!this.analyzersNamesDictionary().find(x => x.studioName === name)) {
+                const customAnalyzerEntry = { studioName: name, serverName: name } as analyzerName;
```

Comment:
```
const customAnalyzerEntry: analyzerName = { studioName: name, serverName: name };
```

please avoid `as` operator whenever possible. This is true evil, as the code compiles w/o required props. ;-)

Reply:
done